optaplanner-core/src/main/java/org/optaplanner/core/impl/constructionheuristic/DefaultConstructionHeuristicPhase.java

```diff
@@ -36,6 +38,13 @@
     protected EntityPlacer entityPlacer;
     protected ConstructionHeuristicDecider decider;
 
+    protected final boolean skipBestSolutionCloningInSteps = true;
```

Comment:
Planned future interaction. It should be configurable, but I am not ready to expose it in the configuration just yet...I 'll add a todo.